runtime/onert/core/include/compiler/BackendManager.h

```diff
@@ -64,6 +65,7 @@ class BackendManager
 private:
   std::map<std::string, std::unique_ptr<void, dlhandle_destroy_t>> _handle_map;
   std::map<std::string, std::unique_ptr<backend::Backend, backend_destroy_t>> _gen_map;
+  backend::controlflow::Backend *_controlflow;
```

Comment:
Regardless of this PR, how about adding initialization of class members in `BackendManager.cc`?

Reply:
I will do that right away! Or the static analyzer will complain